examples/distributed-key-value-store/src/main.rs

```diff
@@ -72,15 +72,16 @@ async fn main() -> Result<(), Box<dyn Error>> {
     swarm.behaviour_mut().kademlia.set_mode(Some(Mode::Server));
 
     // Read full lines from stdin
-    let mut stdin = io::BufReader::new(io::stdin()).lines().fuse();
```

Comment:
can we keep `std::futures::select!` and this?

Reply:
@jxs i did as just line following other example implementations with tokios as the following:
chat/src/main.rs:88:    let mut stdin = io::BufReader::new(io::stdin()).lines();
ipfs-private/src/main.rs:172:    let mut stdin = io::BufReader::new(io::stdin()).lines();

but I can try your approach later tonight.